### ISSUE TITLE
using colon in docstring - interpreted as return type

### ISSUE THREAD

**Issue by Ahrak** [daizutabi/mkapi#17]
Using colons in method/function docstring leads to a problem where mkapi interprets this line as return type of a function:

```python
def show_me_the_problem(self):
"""i show you that this: is a problem"""
    pass
```
Screenshot:
![Screenshot from 2020-06-19 07-06-24](https://user-images.githubusercontent.com/1136923/85098626-86851e00-b1fb-11ea-8cf3-c4e735134fea.png)

**Comment by daizutabi** [comment]
I fixed this issue in version 1.0.6:

```python
class A:
    def func(self):
        """this: is not type."""

    @property
    def prop(self):
        """this: is type."""
```

**Comment by Ahrak** [comment]
resolved. thx :+1:
